ovn-org/ovn-kubernetes
# Bz2058394

Change in the node's IP address shall update all relevant node annotations as well as all relevant components so that the node still works.

If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx"

Trivial changes are exempt from following this template.
If your change is non-trivial, please provide the following information:
-->

**- What this PR does and why is it needed**


**- Special notes for reviewers**



**- How to verify it**
Update br-ex / breth0 IP address on the node, and also make sure that the kubelet node IP is updated and that kubelet is restarted.
After both steps, the node's annotations should reflect the IP address on br-ex / breth0 and the status.Addresses internal IP, respectively.

**- Description for the changelog**
